README.md

```diff
@@ -32,24 +32,24 @@ Before you start your migration, you must do all of the following:
 
 *	Set up a Magento 2 system that meets our <a href="http://devdocs.magento.com/guides/v1.0/install-gde/system-requirements.html">system requirements</a>.
 
-	Set up your system using a topology and design that at least matches your existing Magento 1.x system.
+*	Set up your system using a topology and design that at least matches your existing Magento 1.x system.
 
 *	Do not start Magento 2 cron jobs.
 
 *	Back up or <a href="https://dev.mysql.com/doc/refman/5.1/en/mysqldump.html">dump</a> your Magento 2 database as soon after installation as possible.
 
 *	Check that the data migration tool has a network connection to the Magento 1.x and Magento 2 databases.
 
-	Open ports in your firewall so the migration tool can communicate with the databases and so the databases can communicate with each other.
+*	Open ports in your firewall so the migration tool can communicate with the databases and so the databases can communicate with each other.
 
 *	To provide redundancy in the event of unexpected issues, we advise you to replicate your Magento 1.x database.
 
-*	Migrate Magento 1.x extension and custom code to Magento 2.
+*	Migrate Magento 1.x extensions and custom code to Magento 2.
 
-	Reach out to your extension providers to see if they have been ported yet.
+*	Reach out to your extension providers to see if they have been ported yet.
 
 ## Install the Data Migration Tool
-This section discusses how to install the Magento Data Migration Tool. You can install it from either repo.magento.com or from a GitHub repository.
+This section discusses how to install the **Magento 2 Data Migration Tool**. You can install it from either repo.magento.com or from a GitHub repository.
```

Comment:
1. We usually don't emphasize names with bold

2. The name of the tool does not include "2", it's just Magento Data Migration Tool.
If we had multiple Magento versions (like 3, 4, etc. - hopefully!) it would make sense to put version in the name. For now, we don't.